src/net_processing.cpp

```diff
@@ -3855,11 +3854,7 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
                 if (fSendTrickle && pto->m_tx_relay->fSendMempool) {
                     auto vtxinfo = m_mempool.infoAll();
                     pto->m_tx_relay->fSendMempool = false;
-                    CFeeRate filterrate;
-                    {
-                        LOCK(pto->m_tx_relay->cs_feeFilter);
-                        filterrate = CFeeRate(pto->m_tx_relay->minFeeFilter);
-                    }
+                    const CFeeRate filterrate{pto->m_tx_relay->minFeeFilter.load()};
```

Comment:
Alternative is `const CFeeRate filterrate{CAmount(pto->m_tx_relay->minFeeFilter)}` but I prefer being explicit with atomic load.

Reply:
For consistency, could be used both `load()` and `store()`, or none of them?

Reply:
Going to leave as is